### PR TITLE
Refined Readme.md

### PR DESCRIPTION
**Pull request by nesterone**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,116 +1,105 @@
...
+```
+faf-tool
+mv settings.json.example settings.json
+```
 
-#### update-init
-Update and initialize selected modules: svn up && npm install && npm prune && grunt init
+#### Create and prepare modules for commit
 
-#### checkout-full
```
Comment (sprilukin) [inline Readme.md]: missing in pull request

**Comment by sprilukin** [comment]
And what about list of available tasks? i do not see it in pull request...

**Comment by Krasnyanskiy** [comment]
Also, it is better to add code highlighting and move comments below. It could help to make the example more expressive.
https://gist.github.com/Krasnyanskiy/4d5e554a17957e5a696e#settingsjson

**Comment by nesterone** [comment]
>>> And what about list of available tasks? i do not see it in pull request...

I think it's not a full reference it's just a readme, for full list of task user can just run faf-setup

or faf-tool help if https://github.com/Jaspersoft/faf-tool/issues/8 would be implemented

**Pull request merged**